Octokit/Models/Response/SearchCommitsResult.cs

```diff
@@ -0,0 +1,16 @@
+﻿using System.Collections.Generic;
+using System.Diagnostics;
+using Octokit.Internal;
+
+namespace Octokit
+{
+    [DebuggerDisplay("{DebuggerDisplay,nq}")]
+    public class SearchCommitsResult : SearchResult<SearchCode>
```

Comment:
You'll need a `SearchCommit` result that matches the shape from here:

https://developer.github.com/v3/search/#example-1